pontoon/base/views.py

```diff
@@ -1,3 +1,4 @@
+from datetime import datetime
```

Comment:
This line should go under `from bulk_update.helper import bulk_update`.

We keep `import` statements on top, and only then follow `from X import Y as Z`statements.